CSP_Minizinc_Code.py

```diff
@@ -95,49 +99,54 @@ def select_safety_monitor(self):
     def update_repository(self):
         self.__repo_image.update_current_safety_monitor(self.__selected_safety_monitor)
 
+    #Method for returning the selected safety monitor for performing unit test
+    def get_safety_monitor_unittest(self):
+        return self.__selected_safety_monitor
+
 
 #Contains definition for selecting the platform based on the selected safety monitor. The selected platform information is updated in the repository.

```

Comment:
Is it possible to use a better commenting style? Please refer some code bases. TensorFlow for example